### PR TITLE
Fixed Oclusion in winter level

### PR DESCRIPTION
**Pull request by wbeltran777**
- Fixed occlusion bug. Basically the road dividers had a super tall collider on them to stop people from going out of bounds. This gets in the way of the ray casting that activates the CullSphere. Just made them a bit shorter

### CODE REVIEW

**Comment by m4rtint** [comment]
@wbeltran777 would it be better (cover all cases) to tag those tall walls as a certain layermask? rather than hoping the walls aren't "too tall" in level design? 

and then in the raycast ignore those masks?

**Comment by wbeltran777** [comment]
> @wbeltran777 would it be better (cover all cases) to tag those tall walls as a certain layermask? rather than hoping the walls aren't "too tall" in level design?
> 
> and then in the raycast ignore those masks?

Hmm yeah you're super right, I'll update the PR with those changes

**Comment by mikeslgoh** [comment]
[(1) Ice wall (bottom map) no transparent/zoom feature](https://trello.com/c/UyjzZjT8/586-1-ice-wall-bottom-map-no-transparent-zoom-feature)

File: Assets/_PartyParkade/Script/Game/CameraCulling.cs [hunk 1]
```diff
@@ -41,7 +42,7 @@ private void CameraCullingUpdate()
 
             if (Physics.Raycast(this.transform.position,
                 (m_cullSphere.transform.position - this.transform.position).normalized,
-                out hit))
+                out hit, Mathf.Infinity, layerMask.value))
```
Comment (m4rtint) [inline Assets/_PartyParkade/Script/Game/CameraCulling.cs:45]: if i'm understanding this correctly, you're only casting to the player, and ignoring everything else?

Comment (wbeltran777) [reply Assets/_PartyParkade/Script/Game/CameraCulling.cs:45]: The layermask is set on the prefab as a serialized field. Basically it only checks the 'Obstacle' and 'PlayerModel' layers.

Comment (m4rtint) [reply Assets/_PartyParkade/Script/Game/CameraCulling.cs:45]: interesting

File: Assets/_PartyParkade/Prefabs/Player/Player.prefab [hunk 2]
```diff
@@ -480,6 +483,11 @@ PrefabInstance:
       propertyPath: m_Name
       value: PlayerCar
       objectReference: {fileID: 0}
+    - target: {fileID: 6576194232417591580, guid: b7d96f7a8618dcb4393b1823982ed7d7,
```
Comment (m4rtint) [inline Assets/_PartyParkade/Prefabs/Player/Player.prefab:486]: what did u change here?

File: Assets/_PartyParkade/Script/Game/CameraCulling.cs (hunk 1 above)
Comment (m4rtint) [reply Assets/_PartyParkade/Script/Game/CameraCulling.cs:45]: change it to m_layerMask

File: Assets/_PartyParkade/Prefabs/Player/Player.prefab (hunk 2 above)
Comment (wbeltran777) [reply Assets/_PartyParkade/Prefabs/Player/Player.prefab:486]: Added the 'PlayerModel' layer to the car model. So the ray cast will stop at the car. If it's not there then, if the floor is part of an object, you'll get occlusion when you shouldn't need it.

**Pull request merged**